src/hb-aat-map.hh

```diff
@@ -64,19 +64,24 @@ struct hb_aat_map_builder_t
   {
     hb_aat_layout_feature_type_t  type;
     hb_aat_layout_feature_selector_t  setting;
+    bool is_exclusive;
     unsigned  seq; /* For stable sorting only. */
 
     HB_INTERNAL static int cmp (const void *pa, const void *pb)
     {
       const feature_info_t *a = (const feature_info_t *) pa;
       const feature_info_t *b = (const feature_info_t *) pb;
-      return (a->type != b->type) ? (a->type < b->type ? -1 : 1) :
-	     (a->seq < b->seq ? -1 : a->seq > b->seq ? 1 : 0);
+      if (a->type != b->type) return (a->type < b->type ? -1 : 1);
+      if (a->is_exclusive) return (a->seq < b->seq ? -1 : a->seq > b->seq ? 1 : 0);
+      if ((a->setting & ~1) != (b->setting & ~1)) return (a->setting < b->setting ? -1 : 1);
```

Comment:
Same question here.  Are all non-exclusive settings `> 1`?

Reply:
Same answer -- if the settings differ only in the low bit, they're the on/off selectors for the same feature, so they're "equal" for the purpose of sorting features here.